.clang-format

```diff
@@ -12,4 +12,5 @@ UseTab:          Never
 ColumnLimit:     100
 NamespaceIndentation: Inner
 AlignConsecutiveAssignments: true
+SortIncludes: Never
```

Comment:
I see includes being sorted in this PR.

Reply:
I added that after it had already resorted everything :-\

I was always thinking we would anyway drop the auto-commits and have it rerun those once we have the final `.clang-format` figured out.

Reply:
No more sorting of headers in latests.